src/LuaLib.ts

```diff
@@ -131,6 +133,28 @@ export function getLuaLibModulesInfo(luaTarget: LuaTarget, emitHost: EmitHost):
     return luaLibModulesInfo.get(lualibPath) as LuaLibModulesInfo;
 }
 
+const exportToFeature = new Map<string, ReadonlyMap<string, LuaLibFeature>>();
```

Comment:
Please add a comment saying this is a cache to avoid doing the expensive lookup multiple times, I had to `getLuaLibFeatureForExportName` very carefully before understanding what the point of this variable was.